include/blind.h

```diff
@@ -39,12 +38,27 @@ enum BlindState
     BLIND_SKIPPED
 };
 
+// TODO: Move this to a common interface for other palettes
+typedef struct {
+    const u16* palette;
+    u32 tid;
+    u32 pb;
+} PaletteInfo;
+
+typedef struct
+{
+    enum BlindType type;
```

Comment:
Having the type here isn't necessary. I find it to keep things neat when declaring though.